### ISSUE TITLE
[Feature] Define the year

### ISSUE THREAD

**Issue by Drjacky** [sallar/github-contributions-chart#137]
How to define the year? so, limit the number of screenshots merged into one.

For example just 2023 (or just the current/latest year)

**Comment by Codennnn** [comment]
Or you can try this: https://green-wall.vercel.app/, which is another similar tool that can also meet your needs.

![image](https://github.com/sallar/github-contributions-chart/assets/47730755/2c446e95-4159-46b0-90f6-bf3cdf47ec09)

**Comment by Drjacky** [comment]
@Codennnn Could we also have this as a Github Action? 🙏🏼

**Comment by Codennnn** [comment]
Of course! You can load images through URLs, for example, embedding them in markdown.

```md
![](http://green-wall.vercel.app/api/og/share/Drjacky?year=2020)
```

![](http://green-wall.vercel.app/api/og/share/Drjacky?year=2020)
